Cargo.toml

```diff
@@ -74,6 +74,7 @@ tui = { version = "0.14.0", features = ["crossterm"], default-features = false }
 typed-builder = "0.10.0"
 unicode-segmentation = "1.9.0"
 unicode-width = "0.1.9"
+chrono = "0.4"
```

Comment:
nit: could this be done without adding the `chrono` dependency? I know the vulns are now fixed and all, but if it can just be done with existing dependencies then I would somewhat prefer that.

Reply:
Yes. I used `chrono` because it's convenient, it's otherwise not mandatory.